playwright.config.ts

```diff
@@ -30,22 +30,31 @@ export default defineConfig({
     trace: 'on-first-retry',
   },
 
+  /* Timeout configuration */
+  timeout: 120000,
+
+  /* Configure expect() to use in tests */
+  expect: {
+    /* Setting the timeout for each test */
+    timeout: 120000,
+  },
+
   /* Configure projects for major browsers */
   projects: [
     {
       name: 'chromium',
-      use: { ...devices['Desktop Chrome'] },
-    },
-
-    {
-      name: 'firefox',
-      use: { ...devices['Desktop Firefox'] },
+      use: { ...devices['Desktop Chrome'], ignoreHTTPSErrors: true },
```

Comment:
We must differentiate between HTTPS and HTTP in the common package because Vite demos run without it (only) and the Symfony app with it (only). 

We want to ignore HTTPS errors in Playwright because they cause flaky screenshots. (the same reason we hide vite overlay)